server/src/main/java/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -757,6 +757,15 @@ public PrimaryDataStoreInfo createPool(CreateStoragePoolCmd cmd) throws Resource
             throw new CloudRuntimeException("Failed to add data store: " + e.getMessage(), e);
         }
 
+        // create entry for storage pool capacity
```

Comment:
May I know if this specific to any storage type ?
Because I already see the same code creating capacity entries for cluster scoped storage pool and local storage pools.
cluster storage : https://github.com/apache/cloudstack/blob/75fb1d91eee6e18b2247d39ea4a3cfda04101c07/engine/storage/src/main/java/org/apache/cloudstack/storage/volume/datastore/PrimaryDataStoreHelper.java#L159
Local storage : https://github.com/apache/cloudstack/blob/75fb1d91eee6e18b2247d39ea4a3cfda04101c07/engine/storage/src/main/java/org/apache/cloudstack/storage/volume/datastore/PrimaryDataStoreHelper.java#L152

Reply:
@harikrishna-patnala 
I tested with zone-wide storage pools.
do you think it is better to add `storageMgr.createCapacityEntry(pool.getId());` at line 176 below ?

https://github.com/apache/cloudstack/blob/75fb1d91eee6e18b2247d39ea4a3cfda04101c07/engine/storage/src/main/java/org/apache/cloudstack/storage/volume/datastore/PrimaryDataStoreHelper.java#L176

Reply:
Yes @weizhouapache, that's the only missing piece. Other storage scopes are already handled.

Reply:
@harikrishna-patnala done